### ISSUE TITLE
Create wireframe site

### ISSUE THREAD

**Issue by PirateGrunt** [PirateGrunt/SchoolData#2]
@cassiodorus, @jtf621

A basic wireframe is up. I'm using this template: http://rmarkdown.rstudio.com/rmarkdown_websites.html. We can add  pages as need by creating new .Rmd files. We'll need to manually add the pages to the menu in the file "_site.yml". Otherwise, pretty straightforward.

Site may be found here: http://pirategrunt.com/SchoolData.

**Comment by jtf621** [comment]
@PirateGrunt are you still working on this?

**Comment by cassiodorus** [comment]
Yes but have all new and improved externalized knitr ... back to you soon.

Missed you last night at meetup. Your work on school funding class size etc. would have been (will be) very helpful.

David

**Comment by cassiodorus** [comment]
Note: the Help for pull requests is now at https://help.github.com/articles/creating-a-pull-request/

Ok, 1) I want to share my work, 2) I would be delighted to share my work, 3) I do indeed have a ton of graphics that others might be interested in seeing, and 4) I also have somewhat wandering or speculative commentary to accompany my ever-on-the-move analysis. However, the way I work, and have worked for about three score years, is to actually see someone else, like you or Jeremy, and in just a brief time get new ideas bouncing around in my head. I didn't have to create written commentary that I have complete confidence would soon have to be modified at length. I guess I'm from a different generation. Face to face sessions form my vivid memories. Interaction. The rapid exchange of ideas. I can still recall the faces, the tone of voice, the excitement of insight. If I was building an airplane or a bridge I could think in terms of tasks, but this is exploration. It will soon be time for web site development, for the difficult, creative work on visualization, user interfaces and experiences. Now, I think, is time for words and ideas.

I have the r scripts and I have an rmarkdown file that uses externalization to run the scripts. It is time to add commentary but as you can see by now I find it very frustrating to launch off in yet another direction, to have to keep track of a web site! I have a directory structure. Really simple. My scripts direct graphics to a graphics folder. Can I map this to a web site in some truly simple way? Can a folder be a page? Is there some very low overhead way to continue my development and also, but not instead of, make my work available to others? I am continually fiddling with scripts. It is easy for me to keep track of them but they come and go and have even been known to substantially change their names. If I have to learn new habits, well, I can, but I need to have a context in which I can continue working reliably and comfortably.

David

**Comment by jtf621** [comment]
@PirateGrunt thanks for showing this to me tonight.  Do you want to get together and hack on the data outside of Code for Durham meetups?

**Comment by PirateGrunt** [comment]
Sure! I'll soon have a lot more time to devote to work like this and am looking for an interesting project.

```
  From: jtf621 <notifications@github.com>
```

 To: PirateGrunt/SchoolData SchoolData@noreply.github.com 
Cc: Brian Fannin FanninQED@Yahoo.com; Mention mention@noreply.github.com
 Sent: Tuesday, October 4, 2016 10:42 PM
 Subject: Re: [PirateGrunt/SchoolData] Create wireframe site (#2)

@PirateGrunt thanks for showing this to me tonight. Do you want to get together and hack on the data outside of Code for Durham meetups?—
